Some updates to 2.1.50 (#109)
ankitects/anki-manual: src/preferences.md

## Basic

**Language**  

> Change your display language. You can help to iomprove translations [here](https://translating.ankiweb.net/)

Reviewer: ```suggestion
Change your display language. You can help to improve translations [here](https://translating.ankiweb.net/)
```

Author: Sorry for the mistakes (and thanks for the corrections) :)

**Theme**  

> Dark (night) mode will cause Anki to show cards as white text on a

Reviewer: ```suggestion
Dark (night) mode will make Anki's interface dark, and will cause cards to be shown as white text on a
```

black background. Some card templates may need to be modified to work
properly with this option enabled - please see [night mode
styling](templates/styling.md#night-mode) for more information.

From 2.1.50+ there is an option to switch to day or night mode automatically. 

**Video driver**  
Anki's libraries need a video driver to draw content on the screen.
Due to different hardware and software configurations, the driver that
works best on your machine may vary. ANGLE and OpenGL will perform better
than the software option, but they may not function correctly on some
systems. On Macs, you will almost always want to use the OpenGL option.

> Note: If you're using 2.1.50+ (Qt6), please, check [this](./platform/windows/display-issues.md#qt6)

Reviewer: ```suggestion
Note: If you're using 2.1.50+ (Qt6), please, check [this page](./platform/windows/display-issues.md#qt6).
```

By default, formatting like bold and colors are kept when pasting,
unless the <kbd>Shift</kbd> key is held down. This option reverses the behaviour.

> **Ignore accents in search**

Reviewer: ```suggestion
**Ignore accents in search**

Simple text searches automatically [ignore accents](./searching.md#ignoring-accentscombining-characters).
```

By default, formatting like bold and colors are kept when pasting,
unless the <kbd>Shift</kbd> key is held down. This option reverses the behaviour.

**Ignore accents in search**

> Simple text searches automatically [ignore accents](./searching.md#ignoring-accentscombining-characters).

Reviewer: ```suggestion
When enabled, simple text searches automatically [ignore accents](./searching.md#ignoring-accentscombining-characters).
```

  ask you whether you wish to upload or download. This is useful if
  you have made some changes accidentally, and wish to overwrite them
  with an older version that is on AnkiWeb.

## Backups

> Please, see [this](backups.md#automatic-backups) section of the manual

Reviewer: ```suggestion
Please see [this](backups.md#automatic-backups) section of the manual.
```
